src/main/resources/application.properties

```diff
@@ -1,7 +1,7 @@
 server.port=8081
 spring.datasource.url=jdbc:oracle:thin:@localhost:1521:XE
 spring.datasource.username=system
-spring.datasource.password=system
+spring.datasource.password=dev_carlos123
```

Comment:
Try that this change doesn't goes through, so it won't change everyone's password.